ansible/runperf.yaml

```diff
@@ -0,0 +1,6 @@
+---
+- hosts: performance-server
+  tasks:
+    - include: shutdown_vms.yaml
```

Comment:
Nitpick: `.yml` is the the extension used in all the ansible docs and examples. Though if I recall correctly it actually doesn't matter, even in roles...